### ISSUE TITLE
Suggestion: Ask for region choices as a part of app deployment workflow

### ISSUE THREAD

**Issue by v1gnesh** [superfly/flyctl#738]
We've seen a handful of cases in the forum where folks have experienced issues due to the implicit backup region choice made automatically.
Latest example: https://community.fly.io/t/i-also-cannot-deploy

Like there's a prompt asking for region when deploying an app, should it also ask what one wants to do about the backup region?
That way, they know what to expect (fewer cases of 'why is my thing running there').

Similarly, if a DB is being deployed, should 
- the workflow look for other apps in their org, and say something if app and DB are not in the same region
- ask which app this relates to

Of course, feel free to remove the 'bug' label and use whatever is appropriate.

**Comment by jsierles** [comment]
Good idea! We'll probably be dropping backup regions shortly until we can rethink it with our new Nomad-free architecture. Then we'll revisit these topics.

**Comment by tomr-a5** [comment]
Thanks for raising this. [#3250](https://github.com/superfly/flyctl/pull/3250) removed the old Nomad launch path that selected backup regions implicitly. Current Postgres launch also uses the app’s launch region and attaches it to that app, so I’m going to close this as obsolete. Please reopen if the latest flyctl still makes an unexpected region choice during launch.
